main.go

```diff
@@ -97,15 +105,31 @@ func main() {
 	service.Use(middleware.ErrorHandler(service, true))
 	service.Use(middleware.Recover())
 
```

Comment:
Instead of "nil" can we call the unauthenticated user something like:
* Guest
* Read-only user
* etc.